scripts/uploads/README.md

```diff
@@ -105,6 +108,11 @@ Example run (production run):
 Rscript merge_and_uncode_rca_uploads.R -c NSCLC -u -a $SYNAPSE_AUTH_TOKEN --production
 ```
 
+Example run (production run) using grs:
+```
+Rscript merge_and_uncode_rca_uploads.R -c NSCLC -u -a $SYNAPSE_AUTH_TOKEN --production --use_grs
```

Comment:
Might need to set `store_true` for use_grs  in the merge_and_uncode_rca_uploads.R.

Reply:
I forgot a `true` haha. merge_and_uncode_rca_uploads.R doesn't use `store_true`